CODEOWNERS

```diff
@@ -210,284 +210,30 @@
 /pkg/api/ @cilium/api
-/pkg/aws/ @cilium/aws
-/pkg/azure/ @cilium/azure
-/pkg/bandwidth/ @cilium/sig-datapath
-/pkg/bgp/ @cilium/sig-k8s @cilium/sig-lb
-/pkg/bpf/ @cilium/sig-datapath
 /pkg/byteorder/ @cilium/sig-datapath @cilium/api
-/pkg/cgroups/ @cilium/sig-datapath
 /pkg/client @cilium/api
-/pkg/clustermesh @cilium/sig-clustermesh
-/pkg/command/ @cilium/cli
-/pkg/completion/ @cilium/proxy
-/pkg/components/ @cilium/sig-agent
-/pkg/controller @cilium/sig-agent
-/pkg/counter @cilium/sig-datapath
-/pkg/datapath/ @cilium/sig-datapath
-/pkg/datapath/linux/config/ @cilium/loader
-/pkg/datapath/linux/ipsec/ @cilium/ipsec
-/pkg/datapath/linux/ipsec/xfrm_collector* @cilium/ipsec @cilium/metrics
-/pkg/datapath/linux/node.go @cilium/ipsec @cilium/sig-datapath
-/pkg/datapath/linux/probes/ @cilium/loader
-/pkg/datapath/linux/requirements.go @cilium/loader
-/pkg/datapath/loader.go @cilium/loader
-/pkg/datapath/loader/ @cilium/loader
-/pkg/datapath/ipcache/ @cilium/ipcache
-/pkg/defaults @cilium/sig-agent
-/pkg/ebpf @cilium/sig-datapath
-/pkg/egressgateway/ @cilium/sig-datapath
-/pkg/elf @cilium/sig-datapath
-/pkg/endpoint/ @cilium/endpoint
-/pkg/endpointmanager/ @cilium/endpoint
-/pkg/envoy/ @cilium/proxy
-/pkg/fqdn/ @cilium/sig-agent @cilium/proxy
-/pkg/health/ @cilium/health
-/pkg/hubble/ @cilium/sig-hubble
 /pkg/hubble/metrics @cilium/sig-hubble @cilium/sig-hubble-api
-/pkg/identity @cilium/sig-policy
-/pkg/ipam/ @cilium/sig-ipam
-/pkg/ipam/allocator/alibabacloud/ @cilium/sig-ipam @cilium/alibabacloud
-/pkg/ipam/allocator/aws/ @cilium/sig-ipam @cilium/aws
-/pkg/ipam/allocator/azure/ @cilium/sig-ipam @cilium/azure
-/pkg/ipam/allocator/clusterpool/ @cilium/sig-ipam @cilium/operator
-/pkg/ipcache/ @cilium/ipcache
-/pkg/ipmasq @cilium/sig-agent
-/pkg/k8s/ @cilium/sig-k8s
-/pkg/k8s/apis/cilium.io/client/crds/v2/ @cilium/sig-k8s
-/pkg/k8s/apis/cilium.io/v2/ @cilium/api @cilium/sig-k8s
-/pkg/kafka/ @cilium/proxy
-/pkg/kvstore/ @cilium/kvstore
 /pkg/labels @cilium/sig-policy @cilium/api
-/pkg/launcher @cilium/sig-agent
-/pkg/loadbalancer @cilium/sig-lb
-/pkg/lock @cilium/sig-agent
-/pkg/logging/ @cilium/cli
-/pkg/mac @cilium/sig-datapath
-/pkg/maglev @cilium/sig-lb
-/pkg/maps/ @cilium/sig-datapath
-/pkg/mcastmanager @cilium/sig-datapath
-/pkg/metrics @cilium/metrics
-/pkg/monitor @cilium/sig-datapath
 /pkg/monitor/api @cilium/api @cilium/sig-datapath
-/pkg/monitor/format @cilium/cli @cilium/sig-datapath
 /pkg/monitor/payload @cilium/api @cilium/sig-datapath
-/pkg/mountinfo @cilium/sig-datapath
-/pkg/mtu @cilium/sig-datapath
-/pkg/multicast @cilium/sig-datapath
-/pkg/node @cilium/sig-agent
-/pkg/nodediscovery/ @cilium/sig-agent
-/pkg/option @cilium/sig-agent @cilium/cli
-/pkg/pidfile @cilium/sig-agent
-/pkg/policy @cilium/sig-policy
+/pkg/k8s/apis/cilium.io/v2/ @cilium/api
+/pkg/datapath/linux/ipsec/xfrm_collector* @cilium/metrics
+/pkg/k8s/client/clientset/versioned/ @cilium/api
+/pkg/k8s/client/informers/ @cilium/api
+/pkg/labels @cilium/policy @cilium/api
```

Comment:
fyi, this line here is failing some kind of check.

Reply:
@cilium/policy is now @cilium/sig-policy